docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  web:
```

Comment:
Consider adding your DB as another service in here to be able to use single "docker-compose up" for the entire setup of your project (hint: that would be just a postgres image from Docker Hub)